src/components/AletheiaMenu.tsx

```diff
@@ -104,6 +104,7 @@ const AletheiaMenu = () => {
             </Menu.Item>
             {hasSession && <Menu.Item
                 data-cy={"testLogout"}
+                key="/login"
```

Comment:
Maybe it's better to redirect to the home page, what do you think?

Reply:
You're right, it's better.